### ISSUE TITLE
Lightning Talk Copy

### ISSUE THREAD

**Issue by jefftriplett** [djangocon/djangocon-us-docs#23]
We need copy for the Lightning Talk section of the website. Last year we only had Lightning Talks on the last day of the conference for an hour and each talk had a firm five minute cap with no Q&A. I created a pretty basic Google Doc form for collecting info which I can share with you too. I have 30 minutes assigned for day two and a 60 minute block assigned to day three but I'm open to expanding day two if you believe there is interest.

Prior Art (for ideas):

- http://2014.djangocon.us/lightning_talks_open_spaces/
- http://2011.djangocon.us/lightning/
- https://docs.google.com/forms/d/1S67bNJ0MiWseDIyhtPZzTlmb0o8FbqyoR7AwWV8_CcM/viewform?formkey=dG0yZS1vUVNaNXF4MWdqVk9FbF8yRXc6MQ&hl=en#gid=0

**Comment by chefscott** [comment]
@jefftriplett do you want this as a separate issue or merged with #17 ?

**Comment by jefftriplett** [comment]
Whichever you prefer is fine. I tend to do smaller tasks just because they are easy to mark as done. I haven't played with github's checkbox feature though.

**Comment by chefscott** [comment]
Hey @jefftriplett what do you think about this:

> *Lightening talks are 5 minute talks given on any Django-related topic. They're intended to be low stress and lots of fun -- great if you want to share about a small project or if you're new to speaking.*

> *We are currently accepting, reviewing, and selecting proposals. Sign up here.*

This would link to a Google Doc form or the form could be embedded directly on the page, and it would ask for:
* name
* talk title
* talk description
* email address
* phone number.

**Comment by jefftriplett** [comment]
Looks good. I don't know that we need a phone number but if you ask for it, I'd suggest making it optional. I will send you my spreadsheet and form from last year as a reference too for idea. We ended up being long on time so we let three or four people who were not on the list get up and speak.

**Comment by chefscott** [comment]
I'd prefer to have a phone number to get in contact with people if they are running late or whatever. Email isn't the quickest way to get in touch. On the form I can put a note on why it's there if that's better.

**Comment by jefftriplett** [comment]
Invite sent. I'm not sure if you can see the form so here is a screen shot of the Edit form option.

![](https://www.evernote.com/shard/s27/sh/9ff234bd-8282-4955-b284-cd82124d77ab/c103c580d3e5f616b6d08378bea78fe9/deep/0/DjangoCon-2014--Lightning-Talks---Google-Forms.png)

**Comment by chefscott** [comment]
Here is a link to the website: https://docs.google.com/forms/d/1nw7FntRikapchq1hs7ZqeDhE6wQufph_SxDrju4DaEU/viewform

This is the embed link:
&lt;iframe src="https://docs.google.com/forms/d/1nw7FntRikapchq1hs7ZqeDhE6wQufph_SxDrju4DaEU/viewform?embedded=true" width="760" height="500" frameborder="0" marginheight="0" marginwidth="0"&gt;Loading...&lt;/iframe&gt;

**Comment by ipmb** [comment]
My $0.02...

Don't open sign ups until the conference starts and make it first come
first serve. Less work for organizers and only people that have something
ready will sign up.

**Comment by jefftriplett** [comment]
I agree Peter. To clarify that's what Scott and I discussed. If we decide to add a first (third) day of Lightning talks then we might solicit speakers a week early (for the first day) to prime the pump but that's still up in the air. It doesn't hurt to have the form done early though.

**Comment by chefscott** [comment]
Eh, I wouldn't mind doing the work for higher quality lightening talks and some more solidity.

**Comment by chefscott** [comment]
@jefftriplett Are you okay with the copy? Can we close this out?

**Comment by jefftriplett** [comment]
Hey Scott, your copy looks good. I think that seeing PyCon change their Lightning Talk format to be more upfront and review based in some ways took away what I really liked about them. However, I don't think it ruined them so it's your call. I would like to make one stipulation though that we allow sign-ups during the conference for the third day. I know a lot of people who gave last minute lightning talks over the years at DjangoCon who probably would not have if it were a formal process. If there's not enough interest from conference attendees then we can always used the list as a backup.

**Comment by chefscott** [comment]
Great idea -- I'll get enough talks so I feel we're good for the first one or two days and then still leave it open for the second or third days.

**Comment by jefftriplett** [comment]
I opened this ticket after trying to view the page: https://github.com/djangocon/djangocon.us/issues/46
